lib/customer.rb

```diff
@@ -0,0 +1,29 @@
+require 'csv'
+require 'awesome_print'
+# Customer contains init, processes a CSV into an array, and has a find method
+class Customer
+  attr_accessor(:email, :address, :id)
```

Comment:
These don't need to all be accessors, because your tests don't require the user is able to directly change `email`, `address`, and `id` after the `Customer` instance is created.